### DISCUSSION TITLE
Welcome to Restaurant-App Discussions!

### DISCUSSION THREAD

**Discussion by jurabek** [chayxana/Restaurant-App#84] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Zurikito** [comment]
Hello, i met an issue when trying to run the code, can someone help me solve this
![image](https://github.com/chayxana/Restaurant-App/assets/118882723/cb7822d6-7906-4fd0-aff9-c463710805a1)
Thank you

**Reply by jurabek** [reply]
@Zurikito are you still getting the error ?

**Reply by Zurikito** [reply]
The folder services were missing. So I copy that folder from the outer folder into Docker folder. Deploy successfully but almost all of the services have the same problem "404 not found". Maybe something went wrong right?

**Reply by jurabek** [reply]
I think this is something related to Windows, regarding the endpoints try to use our [postman collection](https://github.com/chayxana/Restaurant-App/tree/develop/postman-collections) and try checkout endpont.

**Reply by jurabek** [reply]
found the issue it actaully docker-compose related thing, starting from new versions of docker-compose now you need to set `-project-directory . `

e.g 
```
docker-compose -f docker/docker-compose.yml \
    -f docker/docker-compose.override.yml \
    -f docker/docker-compose.traefik.yml \
    -f docker/docker-compose.kafka.yml \
    -f docker/docker-compose.grafana.yaml \
    -f docker/docker-compose.otel.yml --project-directory . 
```

**Comment by Zurikito** [comment]
Sorry for my late respone, I did not see your mail in noti. I will take
your advice, thank you

Vào 6:13, Th 7, 16 thg 9, 2023 Jurabek ***@***.***> đã viết:
